glnx-errors.h

```diff
@@ -35,19 +35,7 @@ G_BEGIN_DECLS
  *     return glnx_throw (error, "key must be somevalue, not '%s'", foo);
  * ```
  */
-static inline gboolean G_GNUC_PRINTF (2,3)
```

Comment:
The docstrings should move into the `.c` file now too right?

Reply:
How's that? I've only moved the docstrings for the functions I've moved, but do you also want the (minimal) docstrings for the `_real_` functions, which are also defined in `.c`, moved from `.h`?